ScarletEngine/Sources/Core/Private/IModule.cpp

```diff
@@ -2,9 +2,4 @@
 
 namespace ScarletEngine
```

Comment:
why do we keep the file if it does nothing?

Reply:
It forces the compiler to include the contents of the IModule header. If there are issues in the header and nobody is including it, the first time someone includes it there will be compile issues. In theory every .h file should have a corresponding .cpp file.